packages/slice-machine/pages/_document.js

```diff
@@ -1,22 +0,0 @@
-import Document, { Html, Head, Main, NextScript } from "next/document";
```

Comment:
Are you sure this should be removed?

Reply:
Yes I asked to hugo, this page is a old documentation page not used anymore

Reply:
nice